src/llm/http_llm_calculator.cc

```diff
@@ -46,6 +46,8 @@ namespace mediapipe {
 using chat_entry_t = std::unordered_map<std::string, std::string>;
 using chat_t = std::vector<chat_entry_t>;
 
+#define IGNORE_EOS_MAX_TOKENS_LIMIT 4000
```

Comment:
Is it sufficient? Llama3 has 8k context size for example

Reply:
@mzegla  this is only for scenarios with ignore_eos so for testing purposes.